pyt/cfg/stmt_visitor.py

```diff
@@ -657,6 +657,10 @@ def add_blackbox_or_builtin_call(self, node, blackbox):
             # `scrypt.outer(scrypt.inner(image_name), scrypt.other_inner(image_name))`
             last_return_value_of_nested_call.connect(call_node)
 
+        if '.' in call_function_label:
+            # taint is a RHS variable of taint.lower()
+            rhs_vars.append(call_function_label.split('.', 1)[0])
```

Comment:
Can you try deleting the 

```python
        if isinstance(call, BBorBInode):
            # Necessary to know e.g.
            # `image_name = image_name.replace('..', '')`
            # is a reassignment.
            vars_visitor = VarsVisitor()
            vars_visitor.visit(ast_node.value)
            call.right_hand_side_variables.extend(vars_visitor.result)
```
in https://github.com/python-security/pyt/blob/master/pyt/cfg/stmt_visitor.py#L475-L481

This PR should make that code redundant 👍 

If my hunch is correct about that, can you update this comment
https://github.com/python-security/pyt/blob/master/pyt/cfg/stmt_visitor.py#L670

Reply:
Yep that works. Thanks :+1: